src/clib/lib/job_queue/lsf_driver.cpp

```diff
@@ -735,15 +735,18 @@ void lsf_driver_kill_job(void *_driver, void *_job) {
         char **argv = (char **)calloc(2, sizeof *argv);
         CHECK_ALLOC(argv);
         argv[0] = driver->remote_lsf_server;
-        argv[1] = saprintf("%s %s", driver->bkill_cmd, job->lsf_jobnr_char);
+        argv[1] = saprintf("%s %s %s", driver->bkill_cmd, "-s SIGTERM",
+                           job->lsf_jobnr_char);
 
         spawn_blocking(driver->rsh_cmd, 2, (const char **)argv, NULL, NULL);
 
         free(argv[1]);
         free(argv);
     } else if (driver->submit_method == LSF_SUBMIT_LOCAL_SHELL) {
-        spawn_blocking(driver->bkill_cmd, 1,
-                       (const char **)&job->lsf_jobnr_char, NULL, NULL);
+        spawn_blocking(
+            driver->bkill_cmd, 1,
+            (const char **)saprintf("%s %s", "-s SIGTERM", job->lsf_jobnr_char),
```

Comment:
I am willing to bet that this won't work (do not assume there are automated tests hitting this line).  `saprintf` is probably returning a `const char *` and not `const char **`.